src/config.json

```
@@ -67475,6 +67475,9 @@
     "claiming.optimisn.io",
     "claiming.arbitrhm.foundation",
     "mail-dmail.com",
-    "mr-intl.com"
+    "mr-intl.com",
+    "my.dropz.xyz",
+    "patak.me",
+    "apyeth.eu",
```

Last item cannot have trailing common. It will make this file to invalid json format
```suggestion
    "apyeth.eu"
```